backend/coreapp/views/scratch.py

```diff
@@ -406,12 +406,15 @@ def claim(self, request, pk):
         return Response({"success": True})
 
     @action(detail=True, methods=["POST"])
-    def fork(self, request, pk):
+    def fork(self, request: Request, pk: str) -> Response:
         parent: Scratch = self.get_object()
 
-        request_data = (
-            request.data.dict() if isinstance(request.data, QueryDict) else request.data
-        )
+        # TODO Needed for test_fork_scratch test?
+        if isinstance(request.data, QueryDict):  # type: ignore
```

Comment:
this whole thing seems wrong too

Reply:
IIRC when testing, `request.data` was a QueryDict, not a real dict, even though the stubs said it was supposed to be a dict, and the `**request_data` on L419 didn't work with the QueryDict

Reply:
yeah I saw the same thing. I'm just confused why that's the case..

Reply:
[it appears that DRF varies the type based on what type of data is POSTed, which is gross](https://stackoverflow.com/a/70230790)